tests/TestCase/Error/DebuggerTest.php

```diff
@@ -310,14 +311,14 @@ public function testExportVar()
 
         $result = Debugger::exportVar($View);
         $expected = <<<TEXT
-object(Cake\View\View) {
-	Html => object(Cake\View\Helper\HtmlHelper) {}
-	Form => object(Cake\View\Helper\FormHelper) {}
+object(Cake\View\View) #0 {
```

Comment:
The `#` makes it feel like a count/depth marker not an id marker. Not sure if an `ID` prefix would look better or not.

Reply:
I can do `id:0` if you think it would be clearer.

Reply:
Yea, that is what I was thinking.